java/ql/src/Security/CWE/CWE-614/InsecureCookie.ql

```diff
@@ -26,11 +26,19 @@ predicate isSafeSecureCookieSetting(Expr e) {
   )
 }
 
-from MethodAccess add
+class SecureCookieConfiguration extends DataFlow::Configuration {
+  SecureCookieConfiguration() { this = "SecureCookieConfiguration" }
+
+  override predicate isSource(DataFlow::Node source) { any() }
+
+  override predicate isSink(DataFlow::Node sink) { any() }
```

Comment:
This will have very bad performance in real codebases. Could you restrict the sources to be maybe the qualifier of `setSecure`, or something similar where you'd expect the interesting paths to begin, and the sinks to be the first argument of the `addCookie` call? That should help a lot.